src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -96,6 +96,15 @@ class WorkspaceMembersPage extends React.Component {
             this.validate();
         }
 
+        if (!_.isEqual(prevProps.policyMemberList, this.props.policyMemberList)) {
```

Comment:
I don't think we need deep comparison here (`_.isEqual`). Just a shallow comparison will do it.